framework/src/mysql.js

```diff
@@ -243,9 +243,9 @@ const getTableInstance = async (tableName, tableConfig, connEndpoint = CONN_ENDP
 			const { property, values } = params.whereJsonSupersetOf;
 			query.where(function () {
 				const [val0, ...remValues] = Array.isArray(values) ? values : [values];
-				this.whereJsonSupersetOf(property, val0);
+				this.whereJsonSupersetOf(property, [val0]);
```

Comment:
Can we rename `property` to `column` in respect to knex api?

Reply:
This update will induce a lot of changes within the entire codebase. Let's skip this for now. We can look into this later.

Reply:
We can just rename it here as atleast. When I was reading it, got an wrong idea thinking `property` is a key in the json. Later found it is actually the column name reading the knex doc